XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -42,6 +47,78 @@ private void InitializeData()
+
+            ObservableCollection<NestedCategory> nestedCategories = new ObservableCollection<NestedCategory>();
+            nestedCategories.Add(
+                new NestedCategory("Fruits",  new ObservableCollection<string>{
+                                                            "Apricots",
+                                                            "Bananas",
+                                                            "Grapes",
+                                                            "Strawberries",
+                                                            "Watermelon",
+                                                            "Plums",
+                                                            "Blueberries"
+                }));
+
+            nestedCategories.Add(
+                new NestedCategory("Vegetables", new ObservableCollection<string>{
+                                                            "Broccoli",
+                                                            "Spinach",
+                                                            "Sweet potato",
+                                                            "Cauliflower",
+                                                            "Onion",
+                                                            "Brussel sprouts",
+                                                            "Carrots"
+                }));
+
+            nestedCategories.Add(
+                new NestedCategory("Grains", new ObservableCollection<string>{
+                                                            "Rice",
+                                                            "Quinoa",
+                                                            "Pasta",
+                                                            "Bread",
+                                                            "Farro",
+                                                            "Oats",
+                                                            "Barley"
+                }));
+
+            nestedCategories.Add(
+                new NestedCategory("Proteins", new ObservableCollection<string>{
+                                                            "Steak",
+                                                            "Chicken",
+                                                            "Tofu",
+                                                            "Salmon",
+                                                            "Pork",
+                                                            "Chickpeas",
+                                                            "Eggs"
+                }));
+
+            outerRepeater.ItemsSource = nestedCategories;
+
+            // Intitalize code sample for non-interactive items layout
+            displayLayout.Value = "<muxc:StackLayout x:Name='VerticalStackLayout' Orientation='Vertical' Spacing='8'/>";
```

Comment:
This is a strange mixing of xaml in the codebehind. Why is this here? Generally folks will keep their XAML in the .xaml file to help enable MVVM scenarios and ensure that concerns are separated. Same with the below.

Reply:
So this is the way that @YuliKl used in the NavigationView pages to dynamically update the sample code section based on buttons that were clicked on the page. Since this needs more logic than the '$()' notation allows within the ControlExample.Xaml, this seems like the best way to do it.

Reply:
Oh! I think I misunderstood the comment and intent. To clarify: these chunks of XAML are just to support swapping out the sample code? If so, that's fine.

Reply:
Yep 👍 Should i change the comment to clarify?

Reply:
Please do. Either something like "Set the sample code displayed on the page" or something. When I saw "initialize" and the name of the variable is "displayLayout", I thought we were doing some swapping out of XAML elements.

You could also consider renaming "displayLayout" to something like "sampleCodeLayout" or something that makes it clear that this isn't actually functional code.

Reply:
Done. All your changes should be in place!